ietf/doc/templatetags/ietf_filters.py

```diff
@@ -881,3 +881,19 @@ def badgeify(blob):
             )
 
     return text
+
+@register.filter
+def has_chat_logs(meeting):
+    num = meeting.get_number()
+    return num != None and num >= 60
+
+@register.filter
+def has_recordings(meeting):
+    if meeting.type_id != 'ietf':
+        return True
+    num = meeting.get_number()
+    return num != None and num >= 80
+
+#register.filter
+#define has_notes(meeting):
+    return meeting.uses_notes()
```

Comment:
This looks incomplete - if the intent was to remove the filter L899 should also go.

Reply:
(and I suspect all of these should just be removed once the functionality is moved to Meeting.

Reply:
yes, this was just checking in a snapshot before i got on the plane.  will move to templates and fix shortly.